ScanbotSDKExamples/NET/ReadyToUseUI.iOS/Controller/ImageListController.cs

```diff
@@ -76,29 +86,46 @@ private void OnSaveButtonClick(object sender, EventArgs e)
             var title = "Oops!";
             var body = "Something went wrong with saving your file. Please try again";
 
-            if (!SBSDK.IsLicenseValid())
+            if (!ScanbotSDKGlobal.IsLicenseValid)
             {
                 title = "Oops";
                 body = "Your license has expired";
                 Alert.Show(this, title, body);
                 return;
             }
 
-            var pdf = CreateButton(Texts.save_without_ocr, delegate
+            var pdf = CreateButton(Texts.save_without_ocr, async delegate
             {
                 var output = new NSUrl(nsurl.AbsoluteString + Guid.NewGuid() + ".pdf");
-                SBSDK.CreatePDF(input, output, PDFPageSize.FixedA4);
+                var createPDFConfig = new CreatePDFConfiguration
+                {
+                    InputImageUrls = input,
+                    OutputUrl = output,
+                    PageSize = PDFPageSize.A4
+                };
+
+                await SBSDK.CreatePDF(createPDFConfig);
```

Comment:
@scan-mayank, I think the same point about making an extension method for PDF creation applies here to make sure we don't hide iOS implementation details from the user while still making it easier to use.

Reply:
Implemented Native code in a separate DocumentUtilities file.